src/main/webapp/app/exam/participate/exercises/file-upload/file-upload-exam-submission.component.html

```diff
@@ -1,7 +1,7 @@
 <!--resizable container-->
 <jhi-resizeable-container *ngIf="exercise">
     <!--region Left Panel-->
-    <span left-header *ngIf="exercise.title">{{ exercise.title }}</span>
+    <span class="exercise-title" left-header *ngIf="exercise.title">{{ exercise.title }}</span>
```

Comment:
Why do you use a class and not an id as you did in your other PRs here?

Reply:
The exercises are loaded all on one site and it is not recommended to have the same `id` multiple times within one page:
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Type_Class_and_ID_Selectors#id_selectors

> However, an ID can be used only once per page, and elements can only have a single id value applied to them.

Reply:
In other places we use the exercise id in the html id. But the class also works for me 👍